brightener.h

```diff
@@ -2,11 +2,9 @@
 #include "image.h"
 
 class ImageBrightener {
-private:
-	std::shared_ptr<Image> m_inputImage;
+
 public:
-	ImageBrightener(std::shared_ptr<Image> inputImage);
+	ImageBrightener() = default;
 	bool ValidateImage();
-	int BrightenWholeImage();
-	bool AddBrighteningImage(Image& imageToAdd, int& attenuatedCount);
+    int BrightenWholeImageWithLambda(std::shared_ptr<Image> image);
```

Comment:
Good removal of unused functions.
BTW this is a class without any data members. Is it more suitable to make it a namespace with a function?